1 table.sql

```diff
@@ -0,0 +1,59 @@
+use FlyAirDB
+go
+drop table if exists Booking
+go 
+create table dbo.Booking(
+    BookingId int not null identity primary key,
+    FlightNum char(6) not null 
+        constraint ck_Booking_flight_number_must_start_with_Fly_then_fllowed_by_3_numbers check(FlightNum like 'Fly[0-9][0-9][0-9]'),
```

Comment:
Flight number should not be FLY000

Reply:
You should only have one of each type of constraint on each column. Example if you have multiple check constraints on single column, you should always combine them. That's why sql gives error when you have multiple check constraints. Splitting them with commas makes them multi column constraints.